data/human/news.txt

```diff
@@ -363,7 +396,7 @@ news "housekeeper"
 			`"`
 		word
 			"How is it possible that a man who manages a billion-credit corporation can still be so incompetent or so careless that he routinely misses the toilet bowl when he takes a piss?"
-			"The family I work for offered to let me live in a room in the attic for free, but I'm not sure I want to have that little separation between work and my personal life."
+			"The family I work for offered to let me live in a room in the attic for free, but I'm sure I want to have that little separation between work and my personal life."
```

Comment:
This changes the meaning to be nonsensical

Reply:
Did you see the next commit I made? I somehow changed that without meaning to, but I fixed it.